Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -61,6 +62,7 @@ namespace Platform
 namespace RedirectOutput
 {
     using RedirectOutputFunc = AZStd::function<void(const char*)>;
+    AZStd::vector<AZStd::string> pythonErrorStrings;
```

Comment:
This is a problem.

`AZStd::vector<AZStd::string>` requires it's lifetime to be less than the AZ::SystemAllocator.
This cannot be the case when it is used as a global variable.

This code will crash the ProjectManager on shutdown.

Reply:
Good catch, moved this inside the PythonBindings class so it's allocated along with that.